ui/app/templates/jobs/job/definition.hbs

```diff
@@ -1,6 +1,8 @@
 {{#global-header class="page-header"}}
-  {{#each breadcrumbs as |breadcrumb|}}
-    {{#link-to params=breadcrumb.args class="breadcrumb"}}{{breadcrumb.label}}{{/link-to}}
+  {{#each breadcrumbs as |breadcrumb index|}}
+    <li class="{{if (eq (inc index) breadcrumbs.length) "is-active"}}">
+      {{#link-to params=breadcrumb.args}}{{breadcrumb.label}}{{/link-to}}
```

Comment:
Bulma's breadcrumbs styling makes life pretty difficult, huh?

[I looked at the relevant source](https://github.com/jgthms/bulma/blob/master/sass/components/breadcrumb.sass#L29-L33), and it seems like there's only styling on the `a` child. So here's an untested override idea in case you didn't think of it:

```scss
.breadcrumb a.active {
  @extend .breadcrumb li.is-active a;
}
```

```hbs
<li>
  {{#link-to params=breadcrumb.args}}{{breadcrumb.label}}{{/link-to}}
</li>
```

Reply:
I'll definitely look into this. Generally speaking I try not to override Bulma's markup opinions, but it might be the lesser of two evils in this case.

Reply:
If anything, it might be better to fork it and create a separate class if you want to change the markup. Overriding would make future upgrades annoying. But that might also make switching back to the official component more annoying if you end up wanting to do that in the future.